Contentful/README.md

```diff
@@ -0,0 +1,67 @@
+# Indexing Contentful using the Coveo REST API connector
+This guide explains how you can use the content of the [`SourceJSONConfig.json`](SourceJSONConfig.json) file in a [REST API](https://docs.coveo.com/en/1896/) source to index assets. Your Coveo source will use this JSON configuration to customize HTTP requests for the Contentful Content Delivery API and identify the specific content to extract from the responses.
+
+## Disclaimer
+When [adding a source of content](https://docs.coveo.com/en/3390/index-content/add-or-edit-a-source#add-a-source) in the [Coveo Administration Console](https://docs.coveo.com/en/1841/), Coveo may recommend, or not recommend, using a Coveo [REST API](https://docs.coveo.com/en/1896/) or [GraphQL API](https://docs.coveo.com/en/n6gh2329/) source along with the associated example JSON configuration from this library. Coveo’s recommendation depends on the extent of testing of the system example configuration in proofs of concept.
+
+However, please note that all configurations in this library, including those recommended in the Coveo Administration Console, aren't actively maintained or officially supported. Consider them as starting points that will require customization to fit your specific use case.
+
+## Prerequisites
+To fully understand and effectively use the example JSON configuration, you must:
+- Have a [Coveo organization](https://docs.coveo.com/en/185). Don't have a Coveo organization yet? [Sign up for a free trial](https://www.coveo.com/en/free-trial?utm_marketing_tactic=connectivity_library).
+- Learn about [Coveo connectivity](https://docs.coveo.com/en/1702).
+- Learn [how to configure a REST API source](https://docs.coveo.com/en/1896/).
+
+## Instructions
+1. Get an (Content Delivery) API Key from Contentful for the [Authentication](https://www.contentful.com/developers/docs/references/authentication/). Make sure the API Key has access to the space you want to index.
+
+2. Get the [space id](https://www.contentful.com/help/find-space-id/) from Contentful.
+
+3. To index your Workplace by Facebook content, you will need to [create a REST API source](https://docs.coveo.com/en/1896/).
```

Comment:
I'm not sure why we'd talk about that here? What's the link between both?

Reply:
Wow! Looks like someone copy-pasted the content from the Workplace by Facebook README to get started, but forgot to update the system name thereafter. Nice catch!!!